### ISSUE TITLE
Need for CHANGELOG

### ISSUE THREAD

**Issue by hhzl** [amber-smalltalk/helios#50]
As Amber has [it](https://github.com/amber-smalltalk/amber/blob/master/CHANGELOG). 

What are the changes between

- 0.3.0
- 0.3.1
- 0.3.2
- 0.3.3
- 0.3.4

**Comment by herby** [comment]
You can look at commits, in fact releases were quick and there only few commits for each new release. No breaking changes should be there. Nevertheless, 0.3.4 is needed for 0.14.3, as 0.14.3 change heliosClass to classTag, which 0.3.4 is able to use (0.3.3 breaks). But it is seen in commit messages afaict.

**Comment by hhzl** [comment]
OK, at the moment the most important piece of info I got is that Helios 0.3.4 is needed for Amber 0.14.3.  This should be noted in a file in the repo for everybody to see.

**Comment by herby** [comment]
If you say so. Having the habit update everything in a project, not just amber (eg. running `bower install`) is one of the easy way to never have this problem (0.3.4 was released before 0.14.3 exactly because of that - so no one upgrading amber ended up with incompatible helios).

Hannes Hirzel wrote:

**Comment by hhzl** [comment]
I normally upgrade everything as well. So far I have not run into problems but with 0.3.4. I do

**Comment by herby** [comment]
Then you probably did not upgrade everything this time... or the caret removal and setting helios to uncareted "0.3.0" means it did not upgrade with `bower install` but downgrade instead...

Hannes Hirzel wrote:

**Comment by hhzl** [comment]
No need for guessing. 
I just did a test with 0.14.2 (fix) for Amber and Helios (``"helios": "^0.3.0"``).  Test was **successful**. 

Please have a look https://github.com/hhzl/Amber-snapsvg-demo/issues/4  (log of test activities) and the code  https://github.com/hhzl/Amber-snapsvg-demo/tree/gh-4

**Comment by hhzl** [comment]
Next thing to try out is https://github.com/hhzl/Amber-snapsvg-demo/issues/3 and record the error message(s).

**Comment by hhzl** [comment]
Retest made on https://github.com/hhzl/Amber-snapsvg-demo/issues/3 was OK.

I understand that if I request for Amber ``^0.14.3``   it goes with Helios ``0.3.4``. And that bower pulls in Helios 0.3.4 if I request for  ``^0.14.3`` though I have ``^0.3.4`` for helios in ``bower.json``. Correct?

**Comment by herby** [comment]
Yes. ^0.14.3 is ">=0.14.3 <0.15.0". If imports are present, at least 0.14.3 is needed so the example can be developed. Though, when you get it anew, ^0.14.1 is same as ^0.14.3, since it gets newest 0.14 version anyway.  It is more a style and communication kind of thing - if you do not change it and someone was using that project as a library and updates it, it can still have its old amber. If you change it, update to at least 0.14.3 is forced. So by changing it, you essentially communicate that at least 0.14.3 is actually needed, by leaving it you communicate that update of amber is not essential (and you keep your diff minimal).

Hannes Hirzel wrote:
